app/subscriber/src/features/home/Home.tsx

```diff
@@ -33,28 +33,32 @@ export const Home: React.FC = () => {
     },
     { findContentWithElasticsearch, storeHomeFilter: storeFilter },
   ] = useContent();
-  const [homeItems, setHomeItems] = React.useState<IContentModel[]>([]);
+  const navigate = useNavigate();
```

Comment:
Keep the organization convention alive.  It's easier for all of us to read the code.

Place dependency hooks first.
Then `React.useState`
Then local variables
Then `React.useEffect` and functions